spark/src/test/scala/org/apache/spark/sql/delta/DeltaSuite.scala

```diff
@@ -1607,6 +1607,37 @@ class DeltaSuite extends QueryTest
     }
   }
 
+  test("support Java8 time object") {
+    withSQLConf(SQLConf.DATETIME_JAVA8API_ENABLED.key -> "true") {
```

Comment:
can you give me a reference to support for this in Spark code? I want to understand how this is tested in Spark.